docs/calicoctl/container.md

```diff
@@ -0,0 +1,180 @@
+
+# User guide for 'calicoctl container' commands
+> NOTE: The `calicoctl container` configuration commands are used specifically 
+> when running Calico with Docker default networking.  These commands should 
```

Comment:
"Docker default (bridge) networking"

I think the parenthetical helps clarify since there isn't a super precise term for this.